spec/basic/expected_conditions_spec.js

```diff
@@ -0,0 +1,167 @@
+var EC = protractor.ExpectedConditions;
+
+describe('expected conditions', function() {
+  beforeEach(function() {
+    browser.get('index.html#/form');
+  });
+
+  it('should have alertIsPresent', function() {
+    var alertIsPresent = EC.alertIsPresent();
+    expect(alertIsPresent.call()).toBe(false);
```

Comment:
Alternatively, EC.func can return a promise instead of a function that returns a promise. The down side is that then wait conditions (which I think will be the primary use case will look like `browser.wait(function(){EC.func(x,y)}, 100)` instead of `browser.wait(EC.func(x,y), 100)`